docs/projects/ad7616_sdz/index.rst

```diff
@@ -0,0 +1,428 @@
+   :widths: 25 20 20 20 15
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000
+   * - adc_reset_n
+     - OUT
+     - 43
+     - 97
+   * - adc_hw_rngsel
+     - OUT
+     - 42:41
+     - 96:95
+   * - adc_os **
+     - IN/OUT/INOUT
+     - 40:38
+     - 94:92
+   * - adc_seq_en
+     - OUT
+     - 37
+     - 91
+   * - adc_burst **
+     - OUT
+     - 36
+     - 90
+     - 114
+   * - adc_chsel
+     - OUT
+     - 35:33
+     - 89:87
+   * - adc_crcen **
+     - OUT
+     - 32
+     - 86
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``**`` instantiated only for SER_PAR_N=1 (serial interface)
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+=============== === ========== ===========
+Instance name   HDL Linux Zynq Actual Zynq
+=============== === ========== ===========
+axi_ad7616_dma  13  57         89
+spi_ad7616 **   12  56         88
+axi_ad7616 *    10  54         87
+=============== === ========== ===========
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for SER_PAR_N=0 (parallel interface)
+   -   ``**`` instantiated only for SER_PAR_N=1 (serial interface)
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here <main:/>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the **\*PROJECT LOCATION WITHIN HDL (EX:
+projects/ad7616_sdz/zed)**\ \* location and run the make command by
+typing in your command prompt:
+
+**Linux/Cygwin/WSL**
+
+**\*Say which is the default configuration that's built when running
+``make``, give examples of running with all parameters and also with
+just one. Say that it will create a folder with the name ... when
+running with the following parameters.**\ \*
+
+.. code-block::
+   :linenos:
+   :emphasize-lines: 2, 6
+
+   user@analog:~$ cd hdl/projects/ad7616_sdz/zed
+   // these are just examples of how to write the *make* command with parameters
+   user@analog:~/hdl/projects/ad7616_sdz/zed$ make parameter1=value
+
+   user@analog:~$ cd hdl/projects/ad7616_sdz/zed
+   // these are just examples of how to write the *make* command with parameters
+   user@analog:~/hdl/projects/ad7616_sdz/zed$ make SER_PAR_N=0
+
+The result of the build, if parameters were used, will be in a folder named
+by the configuration used:
+
+if the following command was run
+
+``make SER_PAR_N=0``
+
+then the folder name will be:
+
+``SERPARN0``
+The name will not exceed the limits of the Operating System of 260 characters.
```

Comment:
An empty line needed after ``SERPARN0``. Also, the next phrase should be deleted.

Reply:
Done.